### PR TITLE
adds scripts to bring admiral UI on RHEL 7

### PR DESCRIPTION
**Pull request by Riturajcse**
https://github.com/Shippable/admiral/issues/2151

- I am able to bring admiral UI on RHEL 7

### CODE REVIEW

File: common/scripts/x86_64/RHEL_7/remote/installDb.sh [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+  echo "installing dependencies"
+  yum check-update || true
+  yum install -y nc
+}
+
+__update_sources() {
+  echo "Updating source repo information"
+
+  ## so that posgres dependency do not resolve to base repo
+  echo "exclude=postgresql*" | tee -a "/etc/yum.repos.d/CentOS-Base.repo"
```
Comment (Bharath92) [inline common/scripts/x86_64/RHEL_7/remote/installDb.sh]: is this ok?

File: common/scripts/x86_64/RHEL_7/remote/installDb.sh [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+}
+
+__install_postgres() {
+  echo "Installing Postgres"
+  sudo rm /etc/yum.repos.d/CentOS-Base.repo
```
Comment (Bharath92) [inline common/scripts/x86_64/RHEL_7/remote/installDb.sh]: is this ok?

File: common/scripts/x86_64/RHEL_7/remote/installMsg.sh [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  yum install -y nc
+  yum install -y wget
+}
+
+install_log_rotate() {
+  yum install -y logrotate
+}
+
+install_rabbitmq() {
+  wget "https://github.com/rabbitmq/erlang-rpm/releases/download/v19.3.6.4/erlang-19.3.6.4-1.el7.centos.x86_64.rpm"
```
Comment (Bharath92) [inline common/scripts/x86_64/RHEL_7/remote/installMsg.sh]: is it ok to get centos package?

File: common/scripts/x86_64/RHEL_7/remote/installWorker.sh [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  pushd /tmp
+  mkdir -p $node_scripts_location
+  curl -o "./node-$RELEASE.tar.gz" $node_s3_location
+  tar -xzf node-$RELEASE.tar.gz -C $node_scripts_location --strip-components=1
+  rm -rf node-$RELEASE.tar.gz
+  popd
+
+  cat $node_scripts_location/lib/logger.sh >> installDockerScript.sh
+  cat $node_scripts_location/lib/headers.sh >> installDockerScript.sh
+  cat $node_scripts_location/initScripts/x86_64/CentOS_7/Docker_17.06.sh >> installDockerScript.sh
```
Comment (Bharath92) [inline common/scripts/x86_64/RHEL_7/remote/installWorker.sh]: RHEL 7?

File: common/scripts/lib/_helpers.sh [hunk 5]
```diff
@@ -34,6 +34,11 @@ __check_os_and_architecture() {
     # it as "Ubuntu" so that will not change here.
     local os=$(echo "$NAME" | cut -f 1 -d ' ')
     local os_version="$VERSION_ID"
+    if [ "$os" == "Red" ]; then
```
Comment (shrivara) [inline common/scripts/lib/_helpers.sh]: Compare the whole string `NAME` and if its "Red Hat..." set to RHEL and skip `os` parsing.

File: common/scripts/lib/_helpers.sh [hunk 6]
```diff
@@ -32,8 +32,14 @@ __check_os_and_architecture() {
     # For CentOS, the OS value is defined as "CentOS Linux". We only want
```
Comment (shrivara) [inline common/scripts/lib/_helpers.sh]: Let's move the comment into the else block.

File: common/scripts/lib/_helpers.sh [hunk 7]
```diff
@@ -32,8 +32,14 @@ __check_os_and_architecture() {
...
     # the first part of it, so ignore everything after a space. Ubuntu defines
     # it as "Ubuntu" so that will not change here.
-    local os=$(echo "$NAME" | cut -f 1 -d ' ')
-    local os_version="$VERSION_ID"
+    if [ "$NAME" == "Red Hat Enterprise Linux Server" ]; then
+      local os="RHEL"
+      local os_version=$(echo "$VERSION_ID" | cut -f 1 -d '.')
```
Comment (shrivara) [inline common/scripts/lib/_helpers.sh]: Add a note on why we are doing this specifically for RHEL.

**Pull request merged**